.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
We can consider this in a new issue since this is just copied over from the other pre-commit but this `.gitignore` file is kind of wild — i.e. we do not need to ignore celery files because we don't use celery. I'm a big fan of paring this down to what is actually relevant to us.

Reply:
How trimmed down do you want this to be? (I agree that it should be smaller.) Do you want it to be as minimal as possible?

Reply:
I prefer it so, do we need more than

```
__pycache__
*.pyc
.venv
```

?

Generally my preference is that people update their user's gitignore if they have bespoke things that aren't related to the repository.

Reply:
I agree. Just was not sure how you all prefer it. And so I just copied what was done in the other astral `pre-commit` repo